python/ql/test/experimental/query-tests/Security/CWE-943/mongoengine_get_db_via_connection_good.py

```diff
@@ -0,0 +1,32 @@
+from flask import Flask, request
```

Comment:
It may make more sense to name this file `python/ql/test/experimental/query-tests/Security/CWE-943/mongoengine_connection_module_get_db_good.py` as well as the others.